src/main/resources/db/migration/V3_2__Add_judge_tables.sql

```diff
@@ -0,0 +1,34 @@
+create table judge_item
+(
+    id                   bigint       not null auto_increment,
+    mission_id           bigint       not null,
+    evaluation_item_id   bigint       not null,
+    test_name            varchar(255) not null,
+    programming_language varchar(255) not null,
+    primary key (id)
+) engine = InnoDB
+  default charset = utf8mb4;
```

Comment:
r: not null이 아니어서 수정해야해요!
```suggestion
(
    id                   bigint       not null auto_increment,
    mission_id           bigint       not null,
    evaluation_item_id   bigint,
    test_name            varchar(255),
    programming_language varchar(255),
    primary key (id)
) engine = InnoDB
  default charset = utf8mb4;
```

Reply:
아 자동채점이 없어도 테이블은 저장되는군요! 수정하겠습니다.